exercises/sheet4/sheet4.tex

```diff
@@ -74,11 +110,79 @@
   \]
   for any polytope $P$, where $p_0 = \frac{1}{\vol(P)}\int_P x\,\text{d}x$ denotes the barycenter of $P$.
   \bigskip\bigskip
-\item Complete the proof of Theorem 9.6 in Ziegler's \emph{Lectures on Polytopes}, possibly referring to \cite{bs-1992}.
+\item Complete the proof of Theorem 9.6 in Ziegler's \emph{Lectures on Polytopes}.
   
 \end{enumerate}
 
-\bibliographystyle{plain}
-\bibliography{sheet4}
+
+
+
+\begin{enumerate}
+
+\item It is not possible, since we cannot obtain $Q^c$ (or $\mathcal L^\downarrow(Q^c)$) just from $Q$, $\pi$ and $c$. \newline For example, take $q = 1$, $Q$ any 1-polytope (any interval), $\pi(x,y) = x$ and $c = (0,1)$, then $\pi^c$ is the identity in $P$, so $Q^c = P$, but we are not given $P$, so it's impossible to get $Q^c$ or just $\mathcal L^\downarrow(Q^c)$.
```

Comment:
And if $P$ is part of the input?

Reply:
Oh, that's a good question. I'll think it on tomorrow's morning.

Reply:
In fact, tomorrow's morning I'll try to answer all the problems and questions you gave us to me.

Reply:
I suppose you can get de lower faces of Q^c just with a brute force algorithm because if you are given $P$, then you get Q.

Reply:
Sorry, with $P$ you can get Q^c, not Q.